packages/playground/src/__tests__/api/annotationsApi.int.spec.ts

```diff
@@ -178,26 +183,30 @@ describe.skip('Annotations API', () => {
       createdAnnotationIds[0],
     ]);
     const annotation = listResponse[0];
+    const data = {
+      pageNumber: 8,
+      fileRef: { externalId: 'def_file_changed' },
+      textRegion: { xMin: 0, xMax: 0.1, yMin: 0, yMax: 0.2 },
+    };
     const changes: AnnotationChangeById[] = [
       {
         id: annotation.id,
         update: {
           data: {
-            set: {
-              pageNumber: 8,
-              fileRef: { externalId: 'def_file_changed' },
-              textRegion: { xMin: 0, xMax: 0.1, yMin: 0, yMax: 0.2 },
-            },
+            set: data,
           },
         },
       },
     ];
     const updatedResp = await client.annotations.update(changes);
     const updated = updatedResp[0];
-    expect(updated.data).toEqual({
-      pageNumber: 8,
-      fileRef: { externalId: 'def_file_changed', id: null },
-      textRegion: { xMin: 0, xMax: 0.1, yMin: 0, yMax: 0.2 },
-    });
+    expect(updated).toHaveProperty('data');
+    const updated_data: any = updated.data;
```

Comment:
same

Reply:
I think that makes sense to change this variable's name to cammelCase name. Can you change it?

Reply:
Done here https://github.com/cognitedata/cognite-sdk-js/pull/771/commits/59592f87a2f7e43c0867d9409ae8cfe14f1174bb